### PR TITLE
Add new tests for online editor

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://issues.redhat.com/browse/KOGITO-2473

This PR depends on #479. Do not merge before that.

### CODE REVIEW

**Comment by manstis** [comment]
@sjamboro IDK if you've seen this PR?

It might overlap with your work.

File: packages/online-editor/it-tests/integration/NewFileTest.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      // close Decision navigator
+      cy.get("[data-ouia-component-id='expanded-docks-bar-W'] > div > button ").click();
+
+      // open properties panel, check values and close panel
+      cy.get("[data-title='Properties']").click();
+      cy.get("[name$='definitions.nameHolder']").should("have.value", "new-file");
+      cy.get("[data-title='Properties']").click();
+
+      // open Data Types tab and check there is no item
+      cy.get("a:contains('Data Types')").click();
```
Comment (jomarko) [inline packages/online-editor/it-tests/integration/NewFileTest.ts]: As whole file is `data-` attribute oriented, we could apply such locator also here:
```
cy.get("[data-ouia-component-id='Data Types'] a").click();
```

What do you think?

**Review by jomarko — COMMENTED** [review]
Just one comment/question, otherwise looks good to me.

**Review by jomarko — APPROVED** [review]

**Comment by tomasdavidorg** [comment]
@paulovmr @ederign Build is green after https://github.com/kiegroup/kogito-tooling/pull/479 was merged. Could you please review and merge? Thanks 🙂

**Review by paulovmr — APPROVED** [review]

**Pull request merged**